tensorflow_datasets/structured/movielens.py

```diff
@@ -0,0 +1,422 @@
+# Copyright 2020 The TensorFlow Datasets Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""MovieLens dataset."""
+
+from __future__ import absolute_import
```

Comment:
If these are Python 3 files, we don't need the `__future__` imports.

Reply:
I do believe TFDS supports only Python3, so all dataset code should be Python3 files. However, it seems every other dataset code is including the `__future__` imports, so I included them as well. I am not sure if these are required for TFDS.

Perhaps someone from the TFDS team could comment on whether `__future__` imports are needed?

Reply:
I could be wrong, but we already use Python 3-only features like type annotations. Unless TFDS does source transformations to strip those out for Python 2 packaging I think we can assume that this is Python 3 only.

Reply:
Digging some more, the [v2.0.0](https://github.com/tensorflow/datasets/releases/tag/v2.0.0) release is the last one that supports Python 2. These imports can be safely removed.

Reply:
Got it. Removed in dfe803aa3e5f9ddba2d9e26214c612f095ea1b2a.